linter-requirements.txt

```diff
@@ -1,4 +1,4 @@
-bandit==1.7.4
+bandit[toml]==1.7.4
```

Comment:
Cool change, but somewhat unrelated, right?

Reply:
True, I just did it because I wanted to simplify bandit setup in the pre-commit.